starters/default/README.md

```diff
@@ -43,6 +43,14 @@ _Have another more specific idea? You may want to check out our vibrant collecti
 
     Open the `my-default-starter` directory in your code editor of choice and edit `src/pages/index.js`. Save your changes and the browser will update in real time!
 
+## 🚀 Quick start (Gatsby Cloud)
+
+1.  **Create a Gatsby site.**
```

Comment:
Congrats on the first PR!
I'd propose a different structure, let me know what you think!

So since this is only one step the numbered list style feels a bit off. I'd drop the list if not a 2) or 3) should be added in the near-term future.
Also, what about linking to https://www.gatsbyjs.com/cloud/ so something like:

```md
Deploy this starter with one click on [Gatsby Cloud](https://www.gatsbyjs.com/cloud/):

[<img src="https://www.gatsbyjs.com/deploynow.svg" alt="Deploy to Gatsby Cloud">](https://www.gatsbyjs.com/dashboard/deploynow?url=https://github.com/gatsbyjs/gatsby-starter-default)
```